docs/modules/ROOT/pages/index.adoc

```diff
@@ -18,11 +18,21 @@ Comprehensive audit trails:: The textual policy language of Cerbos makes it idea
 Cerbos Policy Decision Point (PDP) is built for modern, containerised microservice environments with support for both x86-64 and ARM64 architectures, comprehensive observability integrations (metrics, distributed tracing), REST and gRPC endpoints, and native GitOps support (CI tooling, push-to-deploy). 
 
 
-.Cerbos workflow
+== Cerbos workflow
 * Author Cerbos policies to define access rules for your resources. Optionally, write unit tests for the policies using the Cerbos DSL. 
 * Compile the policies and run tests using the Cerbos CLI.
 * Follow your standard development process to push the changes to production. (E.g. create pull request, run CI tests, get approval and merge to prod branch)
 * Cerbos will automatically pull the latest commits from the production branch and update the policies in place without requiring a restart. Your changes are now rolled out! 
 
+== Authorization as a Service
+It was an explicit design choice to have Cerbos be a standalone service rather than a library for the following reasons:
```

Comment:
Always use active voice in documentation :)

```suggestion
Cerbos is designed to be deployed as a service rather than a library compiled into an application. This design choice provides several benefits:
```